src/test/Testcafe/repository_tests/login_create_new_repo.js

```diff
@@ -0,0 +1,34 @@
+import { Selector } from 'testcafe';
+import App from '../app';
+import loginPage from '../page-objects/loginPage';
+import landingPage from '../page-objects/landingPage';
+import CommonPage from '../page-objects/common'
+
+app = new App();
+common = new CommonPage();
+landingpage = new landingPage();
+
```

Comment:
You need to instantiate loginPage aswell?
`let loginPage = new LoginPage()`

Reply:
Fixed 😄